4.validation_and_observability/versions.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/bash
+
+if [ -f /etc/os-release ]; then
+	echo ""
+	echo "OS info:"
+	cat /etc/os-release | head -n 4
+fi
+
+echo ""
+echo "Linux family:"
+uname
+
+echo ""
+echo "Linux Kernel version:"
+uname -r
+
+echo ""
+echo "nvidia-smi:"
+which nvidia-smi
+if [ "$?" == "0" ]; then
+	echo ""
+	echo "NVIDIA versions:"
+	nvidia-smi --version | grep DRIVER
+	nvidia-smi --version | grep CUDA
```

Comment:
```suggestion
	nvcc --version | sed -n 's/^.*release \([0-9]\+\.[0-9]\+\).*$/\1/p
```

nvidia smi provices the cuda version support by the nvidia driver not the actual version of cuda